### PR TITLE
refactor(factory-reset): use factory-reset-status file to store the state of the factory reset

### PR DESCRIPTION
**Pull request by JoergZeidler**
- omnect-device-service bumped to 0.21.0

### CODE REVIEW

File: recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset [hunk 1]
```diff
@@ -102,6 +102,14 @@ factory_reset_wipe_custom() {
     return 0
 }
 
+get_factory_reset_status() {
+    echo -n "$(get_bootloader_env_var "${FACTORY_RESET_STATUS_VAR}")"
+}
+
+clear_factory_reset_status() {
```
Comment (mlilien) [inline recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset:109]: get these one-line functions called in multiple occurrences?

**Review by mlilien — COMMENTED** [review]
imho: description should mention version bump of omnect-device-service

File: recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset (hunk 1 above)
Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset:109]: no, right now only in omnect-device-service-setup.

Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset:109]: but i like to reuse the variable from the factory-reset modul

Comment (mlilien) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset:109]: > but i like to reuse the variable from the factory-reset modul

please elaborate. what does that to do with the usage of a one-line function?

**Review by mlilien — COMMENTED** [review]
https://github.com/omnect/meta-omnect/pull/477#discussion_r1696372070

File: recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset (hunk 1 above)
Comment (JoergZeidler) [reply recipes-omnect/initrdscripts/omnect-os-initramfs/factory-reset]: changed: removed function and used factory-reset variable direct in omnect-device-service-setup

**Review by mlilien — APPROVED** [review]

**Pull request merged**